mass2/core/channels.py

```diff
@@ -672,7 +673,7 @@ def save_analysis(self, path: Path | str, overwrite: bool = False) -> None:
             path = path.with_suffix(".zip")
 
         if os.path.exists(path) and not overwrite:
-            raise ValueError(f"Set save_analysis(...overwrite=True) to overwrite the existing {path=}")
+            raise ValueError(f"File exists; use `save_analysis(...overwrite=True)` to overwrite the existing {path=}")
 
         def drop_columns(df: pl.DataFrame, *columns: str) -> pl.DataFrame:
```

Comment:
This function shouldn't exist it replicates existing polars functionality, instead do `df=df.select(pl.exclude(columns))`